app/controllers/ems_common.rb

```diff
@@ -775,13 +834,6 @@ def retrieve_container_security_protocols
      [_('SSL without validation'), 'ssl-without-validation']]
   end
 
-  def retrieve_hawkular_security_protocols
```

Comment:
I believe this was changes in the master branch and came up with a merge made. But this is fixed in the next commits:

https://github.com/lenovo/manageiq-ui-classic/blob/physical_server_page/app/controllers/ems_common.rb#L776